web/lifecast_res/FthetaMesh.js

```diff
@@ -0,0 +1,251 @@
+import {
+    LDI2_fthetaBgFragmentShader,
+    LDI2_fthetaBgVertexShader,
+    LDI2_fthetaFgFragmentShader,
+    LDI2_fthetaFgVertexShader,
+    LDI3_fthetaBgFragmentShader,
+    LDI3_fthetaBgVertexShader,
+    LDI3_fthetaFgFragmentShader,
+    LDI3_fthetaFgVertexShader
+} from "./LifecastVideoPlayerShaders9.js";
+import * as THREE from './three149.module.min.js';
+
+export const FTHETA_UNIFORM_ROTATION_BUFFER_SIZE = 60; // This MUST match the uniform array size in fgVertexShader
+
+
+/**
+ * That class is a THREE Object3D displaying an LDI.
+ * Thus it should contains all code related with the LDI geometry and material (texture)
+ * It should not contain code related with the player (pause, play action...; mouse interaction; scene creation, camera/vr creation)
+ * TODO
```

Comment:
Notes on the comment here:
* The documentation should describe the class as it is now, less about how it should be in the future.
* I prefer to keep the TODO list outside the code and in the PR comments
* Instead of formatting comment like this:
```
/*
* line
*/
```
Please omit the * on the middle lines, i.e.
```
/*
line
*/
```